capture/exception.ts

```diff
@@ -0,0 +1,12 @@
+import raven from 'raven'
+import config from '../config'
+
+raven.config(config.get('sentry.dsn'), {
+  environment: config.get('node.env')
+}).install()
+
+const captureException = (error: Error): void => {
+  if (error.message !== 'body not valid') raven.captureException(error)
```

Comment:
maybe message 'body not valid` make collection like `lang` or `message.json` so, if sentralize config this msg can reusable 👍

Reply:
this is what I mean

Reply:
Nah itu wkwkw

Reply:
that's true boys,,, 😆

Reply:
updated yes df25ea7